com.termius.Termius.yaml

```diff
@@ -0,0 +1,69 @@
+id: com.termius.Termius
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: termius
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
either this support wayland out of the box so it's `fallback-x11` or it doesn't and in that case `wayland` should be removed.

Reply:
It doesn't work out of the box, I will remove wayland.